dap_prinz_green_jobs/notebooks/README.md

```diff
@@ -14,6 +14,10 @@ The `between_measures_analysis.ipynb` notebook contains analysis of green measur
 
```

Comment:
there was a merge conflict with this file but i edited it very slightly to address !